static/memoradical.css

```diff
@@ -128,3 +128,14 @@ nav button:first-of-type {
 .card.response {
     background-color: #c1ffc1;
 }
+
+.memoradical-help {
+    font-family: system-ui;
```

Comment:
I might suggest that we should instead just set `font-family` on `html` to unify the font across the entire application?